get_nxpRdLib.sh

```diff
@@ -6,72 +6,38 @@ NC='\033[0m' # No Color
 prereq() {
   echo -e "[${blue}Installing prerequisites${NC}]"
   sudo apt-get update
-  sudo apt-get -y install libglib2.0 glib-networking-services libreadline6-dev libglib2.0-dev glib-networking-services python2.7-dev build-essential autoconf libtool
+  sudo apt-get -y install python2.7-dev build-essential cmake
 }
 
-neardal() {
-  echo -e "[${blue}Downloading NeardAL${NC}]"
-  git clone https://github.com/svvitale/neardal
-  cd neardal
 
-  echo -e "[${blue}Running autogen.sh${NC}]"
-  ./autogen.sh
-
-  echo -e "[${blue}Building NeardAL${NC}]"
-  make
-
-  echo -e "[${blue}Installing NeardAL${NC}]"
-  sudo make install
-
-  cd ..
-}
-
-wiringpi() {
-  echo -e "[${blue}Downloading wiringPi${NC}]"
-  git clone git://git.drogon.net/wiringPi
-  git reset --hard 72b2af231be337d45a557ce6ca3ed1eeb6675ffd
-
-  echo -e "[${blue}Building and Installing wiringPi${NC}]"
-  cd wiringPi
-  ./build
-
-  echo -e "[${blue}Running ldconfig${NC}]"
-  sudo ldconfig
-
-  cd ..
-}
 
 nxp() {
-  echo -e "[${blue}Downloading NXP Reader Library${NC}]"
-  wget http://www.nxp.com/redirect/explore-nfc-dev_latest -O nxp.zip
+echo -e "[${blue}Downloading NXP Reader Library${NC}]"
+  wget https://cache.nxp.com/documents/software/sw3693.zip -O nxp.zip
   unzip -o nxp.zip
 
-  tar --overwrite -xzf neard-explorenfc_*.orig.tar.gz
-  cd neard-explorenfc-*
-
-  export WIRINGPI_CFLAGS=-mword-relocations\ -I/usr/local/include
-  export WIRINGPI_LIBS=-L/usr/local/lib\ -lwiringPi
-
-  echo -e "[${blue}Running bootstrap${NC}]"
-  ./bootstrap
-
-  echo -e "[${blue}Configuring NXP Reader Library${NC}]"
-  ./configure --prefix=/usr --sysconfdir=/etc
+value=$(cat NXP_NFC_Reader_Library_licencefile.txt)
+if (whiptail --title "Please read and accept the NXP NFC Reader Libary greement" --yesno --yes-button "accept" --no-button "reject" --scrolltext "$value" 0 0) then
```

Comment:
When installing via pip, the console output is not available to the user and no prompts can be answered.  Would it be possible to include verbage in the README.md that says, "By using this library, you're accepting the terms of the NXP NFC Reader Library Agreement"?